File: typescript_discord_bot_startup_sl8s/app/src/model/example/example_wrapper/example_wrapper.ts
import { BaseModelWrapper } from "@antonpichka/vitruvius";
import Example from "../example";

class ExampleWrapper extends BaseModelWrapper {
    public constructor(listObject: Array<any>) {
        super(listObject);
    }

    public override createModel(): Example {
        return new Example(this.listObject[0]);
    }
}

export default ExampleWrapper;

File: dart_web_startup_sl8s/app/lib/model/example/example_wrapper/example_wrapper.dart
import 'package:dart_web_startup_sl8s/model/example/example.dart';
import 'package:library_architecture_mvvm_modify/library_architecture_mvvm_modify.dart';
import 'package:meta/meta.dart';

@immutable
base class ExampleWrapper extends BaseModelWrapper {
  const ExampleWrapper({required List<dynamic> listObject}) : super(listObject);

  @override
  Example createModel() {
    return Example(uniqueId: listObject[0]);
  }
}